modules/core/src/test/java/org/apache/ignite/internal/processors/cache/IgniteClientCacheStartFailoverTest.java

```diff
@@ -362,15 +360,6 @@ public void testRebalanceStateConcurrentStart() throws Exception {
                                 ", val=" + i + ']', e);
 
                             CacheConfiguration ccfg = cache.getConfiguration(CacheConfiguration.class);
-
-                            TransactionSerializationException txEx = X.cause(e, TransactionSerializationException.class);
-
-                            boolean notContains = !txEx.getMessage().contains(
-                                "Cannot serialize transaction due to write conflict (transaction is marked for rollback)"
```

Comment:
This message is used to throw `IgniteTxSerializationCheckedException` in `GridCacheMapEntry#serializationError`. Both seems to be unused and must be removed.